common/Link.py

```diff
@@ -1,17 +1,8 @@
-class Link():
-    def __init__(self, address_name, categories=""):
-        self.address_name = address_name
-        # Make sure that input is a list
-        if isinstance(categories, list):
-            self.categories = categories
-        else:
-            self.categories = [categories]
+from db import db
 
-    def get_categories(self):
-        return self.categories
 
-    def get_address_name(self):
-        return self.address_name
-
-    def append_new_category(self, category):
-        self.categories.append(category)
+class Link(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    url = db.Column(db.String(50))
+    owner_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    categories = db.Column(db.String(50))
```

Comment:
In the future, categories should save in their own database table, and here we will have only the foreign keys!